source/Textures.js

```diff
@@ -0,0 +1,20 @@
+var Pixi = require("pixi.js")
+
+import Junkership from "./Junkership.js"
+
+exports.initTex = function initTex() {
+    PIXI.loader
+        .add("redJunkership",    require("./images/red-starship.png"))
+        .add("yellowJunkership", require("./images/yellow-starship.png"))
+        .add("greenJunkership",  require("./images/green-starship.png"))
+        .add("blueJunkership",   require("./images/blue-starship.png"))
+        .add("projectile",       require("./images/projectile.png"))
+        .add("trashbot",         require("./images/enemy-starship.png"))
+        .load()
+
+        .on("complete",setup)
+}
+
+function setup() {
+    game.spawnWave()
```

Comment:
Tried that. Spawning the Junkership here caused a slightly different problem. We had one correctly spawning Junkership, but the game checked playerCount before the correctly spawned Junkership existed, so it spawned an incorrectly loaded one anyway (the gameOver() mechanic)